Makefile

```diff
@@ -263,4 +266,5 @@ olm-push: bundle-push index-push
 	generate-manifests \
 	tools \
 	bundle \
-	bundle-build
+	bundle-build \
+	manifests
```

Comment:
Weird, this should have been handled by https://github.com/openshift/kubernetes-nmstate/pull/190

Reply:
Ah yeah. I rebased my branch and it's ok now